en/full-node.md

```diff
@@ -0,0 +1,362 @@
+transmit their transactions to the network and by notifying them when a
+transaction affects their wallet. If not enough nodes perform this
+function, clients won't be able to connect through the peer-to-peer
+network---they'll have to use centralized services instead.
+
+Many people and organizations volunteer to run full nodes using spare
+computing and bandwidth resources---but more volunteers are needed to
+allow Bitcoin to continue to grow.  This document describes how you can
+help and what helping will cost you.
+
+## Costs And Warnings
+
+Running a Bitcoin full node comes with certain costs and can expose you
+to certain risks. This section will explain those costs and risks so you
+can decide whether you're able to help the network.
+
+### Special Cases
+
+Miners, businesses, and privacy-conscious users rely on particular
+behavior from the full nodes they use, so they will often run their own
+full nodes and take special safety precautions. This document does not
+cover those precautions---it only describes running a full node to help
+support the Bitcoin network in general.
+
+Please [consult an expert](/en/community) if you need help setting up
+your full node correctly to handle high-value and privacy-sensitive
+tasks.
+
+### Secure Your Wallet
+
+It's possible and safe to run a full node to support the network and use
+its wallet to store your bitcoins, but you must take the same
+precautions you would when using any Bitcoin wallet.  Please see the
+[securing your wallet page](/en/secure-your-wallet) for more
+information.
+
+### Minimum Requirements
+
+Bitcoin Core full nodes have certain requirements. If you try running a
+node on weak hardware, it may work---but you'll likely spend more time
+dealing with issues. If you can meet the following requirements, you'll
+have an easy-to-use node.
+
+* Desktop or laptop hardware running recent versions of Windows, Mac OS
+  X, or Linux.
+
+* 50 gigabytes of free disk space
+
+* 2 gigabytes of memory (RAM)
+
+* Broadband Internet connection
+
+* 6 hours a day that your full node can be left running. (You can do
+  other things with your computer while running a full node)
+
+### Possible Problems
+
+* **Legal:** Bitcoin use is [prohibited or restricted in some
+  areas](http://bitlegal.io/)
+
+* **Anti-virus:** Several people have placed parts of known computer
+  viruses in the Bitcoin block chain. This block chain data can't infect
+  your computer, but some anti-virus programs quarantine the data any
+  way, making it more difficult to run a full node. This problem mostly
+  affects computers running Windows
+
+* **Attack target:** People who want to disrupt the Bitcoin network may
+  attack full nodes in ways that will affect other things you do with
+  your computer, such as an attack that limits your available download
+  bandwidth or an attack that prevents you from using your full node's
+  wallet for sending transactions
+
```

Comment:
May I suggest adding the following (or similar)

\*\*Bandwidth limit\*\*: Some Internet plans will charge an additional amount for any excess upload bandwidth used that isn't included in the plan. You may want to check if your Internet connection is subjected to such limitations or monitor your bandwidth consumption.